src/services/domain/bulk-operations.service.ts

```
@@ -0,0 +1,233 @@
+    this.validateRepositoryProvider('bulkDeleteByType');
+
+    try {
+      const kuzuClient = await this.getKuzuClient(mcpContext, clientProjectRoot);
+      const warnings: string[] = [];
+      let totalCount = 0;
+      const deletedEntities: Array<{ type: string; id: string; name?: string }> = [];
+
+      if (entityType === 'all') {
+        // Handle deletion of all entity types
+        const entityTypes = ['Component', 'Decision', 'Rule', 'File', 'Context'];
+        
```

ESLint

[error] Delete `········`

(prettier/prettier)